.env

```diff
@@ -1,4 +1,5 @@
 VUE_APP_RPC_URL=http://localhost:26657
 VUE_APP_RPC_WS_URL=ws://localhost:26657
 VUE_APP_RPC_PM_URL=http://localhost:9984
-VUE_APP_NETWORK=Testnet
\ No newline at end of file
+VUE_APP_NETWORK=Testnet
+VUE_APP_CID_RESOLVER=https://cid-resolver.rddl.io
```

Comment:
This should point to a `http://localhost:<SOME_PORT>` address, thus be added to the docker-compose.yaml in https://github.com/rddl-network/node_maintenance/